tests/integration/routes/accounts/patch-accounts-test.js

```diff
@@ -140,7 +141,15 @@ getServer(function (error, server) {
     })
 
     group.test('with ?include=foobar', {todo: true}, function (t) {
-      t.end()
+      var options = _.defaultsDeep({
+        url: '/accounts?include=foobar'
```

Comment:
this should be `url: '/accounts/abc4567?include=foobar'`. And then you need to add query validation to the route definition